source/widgets/onboarding_setup/wizard_pages.py

```diff
@@ -0,0 +1,345 @@
+        self.layout_.addWidget(self.use_file_associations)
+        self.layout_.addWidget(self.explanation_label)
+
+    def evaluate(self):
+        if self.use_file_associations.isChecked():
+            if self.select is not None:  # then we should make a desktop file
+                assert self.select.path is not None
+
+                if self.select.path.is_dir():
+                    pth = self.select.path / get_default_shortcut_destination().name
+                else:
+                    pth = self.select.path
+
+                generate_program_shortcut(pth, exe=str(self.prop_settings.exe_location))
+                return
+
+            if self.platform == "Windows":
+                register_windows_filetypes(exe=str(self.prop_settings.exe_location))
+
+        elif self.platform == "Windows":
+            if self.addtostart.isChecked():
+                generate_program_shortcut(
+                    get_default_shortcut_destination(),
+                    exe=str(self.prop_settings.exe_location),
+                )
+            if self.addtodesk.isChecked():
+                # TODO: Consider using platformdirs to find the exact path
+                typical_paths = [
+                    Path("~/Desktop/Blender Launcher V2").expanduser(),
+                    Path("~/OneDrive/Desktop/Blender Launcher V2").expanduser(),
+                ]
+                exceptions = []
+                for pth in typical_paths:
+                    try:
+                        generate_program_shortcut(
+                            pth,
+                            exe=str(self.prop_settings.exe_location),
+                        )
+                        break
+                    except Exception as e:
+                        exceptions.append(e)
+                if len(exceptions) == len(typical_paths):  # all paths failed to generate
+                    raise Exception("Exceptions raised while generating desktop shortcuts: {exceptions}")
+
+
+TITLEBAR_LABEL_TEXT = """This disables the custom title bar and uses the OS's default titlebar.
+
+In Linux Wayland environments, this is recommended because you will be
+able to use the title for moving and resizing the windows.
+Our main method of moving and resizing works best on X11."""
+
+HIGH_DPI_TEXT = """This enables high DPI scaling for the program.
+automatically scales the user interface based on the monitor's pixel density."""
+
+
+class AppearancePage(BasicOnboardingPage):
+    def __init__(self, prop_settings: PropogatedSettings, parent: BlenderLauncher):
+        super().__init__(prop_settings, parent=parent)
+        self.setTitle("BLV2 appearance")
+        self.setSubTitle("Configure how BLV2 Looks")
+        self.layout_ = QVBoxLayout(self)
+
+        self.titlebar = QCheckBox("Use System Titlebar", self)
+        self.titlebar.setChecked(get_use_system_titlebar())
+        titlebar_label = QLabel(TITLEBAR_LABEL_TEXT, self)
```

Comment:
Implementing the Linux specific message
```suggestion
        if get_platform() == "Linux":
            titlebar_label = QLabel(TITLEBAR_LABEL_TEXT_LINUX, self)
        else:
            titlebar_label = QLabel(TITLEBAR_LABEL_TEXT, self)
```